### PR TITLE
Updating contsistenly the scripts with GiDInterface

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

File: applications/StructuralMechanicsApplication/python_scripts/Kratos_Main_Structural.py [hunk 1]
```diff
@@ -1,110 +1,88 @@
...
-    if( report ):
-        used_time = time_fp - time_ip
-        print("::[KSM Simulation]:: [ %.2f" % round(used_time,2),"s", process," ] ")
-
-#### TIME MONITORING END ####
-
-#### PARSING THE PARAMETERS ####
-
-# Import define_output
+from KratosMultiphysics import *
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/python_scripts/Kratos_Main_Structural.py:3]: Why did you remove the absolute imports? I thought this was the way to go?

Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/python_scripts/Kratos_Main_Structural.py:3]: It supposed to be, but @rubenzorrilla and @RiccardoRossi said that for the main script is not mandatory

Comment (philbucher) [reply applications/StructuralMechanicsApplication/python_scripts/Kratos_Main_Structural.py:3]: wouldn't it make sense since this is what ppl will be using a lot?

Comment (jcotela) [reply applications/StructuralMechanicsApplication/python_scripts/Kratos_Main_Structural.py:3]: @philbucher In terms of absolute imports, we have internally some issues (see the discussion in #42). The point is that we don't want to impose on the user the need to know which application defines which variable (since it is an implementation detail and, in our experience, changes frequently), but this implies the absolute imports, which is less clear than the relative ones.
Until we find a satisfactory solution (i.e. everything except the variables is treated as relative imports, but this is a bit hard to achieve technically), I'd also recommend using absolute imports as in this PR.

Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/python_scripts/Kratos_Main_Structural.py:3]: @jcotela I would say except for utilities of tools auto-contained where we already know where the dependencies are

Comment (philbucher) [reply applications/StructuralMechanicsApplication/python_scripts/Kratos_Main_Structural.py:3]: ok fine for me, just wanted to know
